tests/test_pipelines.py

```diff
@@ -342,7 +346,7 @@ def test_stable_diffusion_k_lms(self):
         image_slice = image[0, -3:, -3:, -1]
 
         assert image.shape == (1, 128, 128, 3)
-        expected_slice = np.array([0.5067, 0.4689, 0.4614, 0.5233, 0.4903, 0.5112, 0.524, 0.5069, 0.4785])
+        expected_slice = np.array([0.5674, 0.512, 0.5586, 0.6235, 0.4321, 0.5654, 0.411, 0.4014, 0.4463])
```

Comment:
That's quite a difference? Do the tests currently fail for you?

Reply:
yes on CPU vs GPU the results are different here...